### ISSUE TITLE
Dark theme doesn't run for forum reply fields.

### ISSUE THREAD

**Issue by VFDan** [VFDan/Scratch-Dark-Theme#9]
![image](https://user-images.githubusercontent.com/44413950/88451777-91992280-ce27-11ea-9cff-42f01a9bab31.png)

Since it is an iframe, https://stackoverflow.com/a/7571138/ might be helpful.

**Comment by MiniCoder11** [comment]
I've partially fixed this. The white background of the main preview field will now be consistent with the main color palette. There still is no easy way to modify the iFrame contents without jQuery due to the Cross-Origin policy.

**Comment by MiniCoder11** [comment]
@VFDan I do not think there is any "proper" way to fix this without bypassing security restrictions using unintended exploits. Scratch users may not feel safe with solutions that expose the contents of their webpages in such ways. Should we close the issue and declare it as "won't fix" for now?

**Comment by VFDan** [comment]
Can't we just do jQuery then, if that's the last option besides exploits? What we should do is put all the styles in seperate documents, then instead of calling every style we just append the document to the page.  https://stackoverflow.com/questions/583753/using-css-to-affect-div-style-inside-iframe/7571138#7571138 is how we can easily add the style to the iframe, and I wouldn't call that an exploit.

**Comment by MiniCoder11** [comment]
We could, but then wouldn't we would be going beyond the limitations of a userscript? Once multiple files become involved, we would need the user to perform the installation of several files to get the dark theme to work. If we are using only one file, then I think we should do the rewrite for #12 first so that the style can apply to iframes by the time they load.

**Comment by VFDan** [comment]
Nope, we don't need the user to install multiple files. We just do direct links to the files.

**Comment by MiniCoder11** [comment]
Okay, sounds good : )
